### ISSUE TITLE
Seller Experience: The description reads "Requires a paid plan" even if user has purchased the Pro plan

### ISSUE THREAD

**Issue by andres-blanco** [Automattic/wp-calypso#62470]
This was reported by @obenland:
<img width="1223" alt="image" src="https://user-images.githubusercontent.com/375980/161326355-7eef8893-0977-4178-a75d-2c17d42623ce.png">
On [WordPress.com](http://wordpress.com/) I went through "Switch Site" and "Add new site" to the onboarding flow.
There I encountered two variants, one where the plan step is first and one where the intent step is first.
On plan step first I selected Pro and then Sell in the intent step after, and arrived at my screenshot from above

I couldn't reproduce. I'll leave this issue open so others can try it.

**Comment by sixhours** [comment]
> There I encountered two variants, one where the plan step is first and one where the intent step is first.
On plan step first I selected Pro and then Sell in the intent step after, and arrived at my screenshot from above

I'm not able to reproduce this one, either, but I have a hunch. I've noticed sometimes when I drop out of the signup/intent flow without completing it, my information (site title, etc.) is still in state and I'm automatically brought back to the step where I dropped off. Maybe Konstantin was dropped into the intent step after having exited it with a free site.
